openstack/smn/v2/logtank/requests.go

```diff
@@ -0,0 +1,64 @@
+package logtank
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+var RequestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json;charset=UTF-8"},
+}
+
+// Ops is a struct that contains all the parameters.
+type Ops struct {
+	LogGroupID string `json:"log_group_id" required:"true"`
+
+	LogStreamID string `json:"log_stream_id" required:"true"`
+}
+
+// Create a logtank with given parameters.
+func Create(client *golangsdk.ServiceClient, topicUrn string, ops Ops) (r CreateResult) {
+	b, err := golangsdk.BuildRequestBody(ops, "")
+	if err != nil {
+		r.Err = err
+		return
+	}
+
+	_, r.Err = client.Post(createURL(client, topicUrn), b, &r.Body, &golangsdk.RequestOpts{
+		OkCodes:     []int{201},
```

Comment:
it's better to delete `OkCodes` to avoid error when the status code changed.

Reply:
maybe all the request need be remove okCodes?

Reply:
yes, we can use default okCodes for most situations